acceptance/bundle/artifacts/nil_artifacts/databricks.yml

```diff
@@ -5,6 +5,6 @@ artifacts:
   # This reproduces the issue from GitHub #3017
   # where artifacts have empty definitions (only comments)
   my_artifact:
-    # build: <first_option_command>
-    # path: <path>
-    # build: <second_option_command>
+  # build: <first_option_command>
+  # path: <path>
+  # build: <second_option_command>
```

Comment:
This is different from what I expected; when I uncomment these lines, they are incorrectly indented.